### PR TITLE
[844] - Added Travis files

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
- Task [844](https://www.quartile.co/web#id=844&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    # by the `pov-wkhtmltopdf` repo.
+    - PHANTOMJS_VERSION="latest"
+    # The above line controls the PhantomJS version that is used for JS testing.
+    #   It is not necessary to include this value unless you are altering the default.
+    #   Use `OS` to skip the PhantomJS upgrade & use the system version instead.
+    - WEBSITE_REPO="1"
+    # Use the above line to install dependencies that are required for website repos:
+    # * SASS & Bootstrap-SASS
+    # * Compass
+    - EXCLUDE="partner_category_hierarchy,website_forum_visibility"
```
Comment (TimLai125) [inline .travis.yml]: @YogeshMahera-SerpentCS Please remove this.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    # tests)
+    #  - TESTS="1.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1"
+    #  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
+    # QRTL: Add following tests the installation of all modules including those are excluded
+    - stage: install
+      env:
+        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1" TEST_ENABLE="0" EXCLUDE=""
+    - stage: install
+      env:
+        - TESTS="1" ODOO_REPO="OCA/OCB" TEST_ENABLE="0" EXCLUDE=""
```
Comment (TimLai125) [inline .travis.yml]: install stage is not needed.

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 Amended Changes

**Comment by YogeshMahera-SerpentCS** [comment]
@TimLai125 @yostashiro Could you please review it?

**Comment by YogeshMahera-SerpentCS** [comment]
@TimLai125 Test Comment

**Pull request merged**